Examples/Segmentation/ShapeDetectionLevelSetFilter.cxx

```diff
@@ -558,30 +576,31 @@ int main( int argc, char *argv[] )
   WriterType::Pointer writer3 = WriterType::New();
   WriterType::Pointer writer4 = WriterType::New();
 
+  const std::string outputImageFilePrefix = outputImageFile.substr( 0, outputImageFile.length() - 4 );
```

Comment:
This line looks like it is intended to remove the file extension and assumes it's "."+ 3 characters. Would using some utility methods like KWSys::GetFilenameWithoutExtension be better?

Reply:
Done!